### ISSUE TITLE
Import and use the colors palette

### ISSUE THREAD

**Issue by afercia** [Yoast/YoastSEO.js#952]
The new colors palette merged in https://github.com/Yoast/yoast-components/pull/109 should be used also here and in other projects.

To do this there's the need to:
- find the best way to import the colors, /cc @atimmer 
- replace all current colors in `YoastSEO.js` with the new ones, this part is almost done on a local branch

**Comment by atimmer** [comment]
We want to move all UI out of this repository to the `yoast-components` repository, so I'd rather to that instead of importing colors here.
